WORKSPACE

```diff
@@ -35,6 +35,12 @@ load("//web:repositories.bzl", "browser_repositories", "web_test_repositories")
 
 web_test_repositories()
 
+git_repository(
+    name = "io_bazel_skydoc",
```

Comment:
Please can you use the non-skydoc version? https://github.com/bazelbuild/skydoc/pull/64 is already set to convert their use into the skylib version, but they've not looked at it yet.

Reply:
Done.